src/nyc_trees/apps/quiz/views.py

```diff
@@ -5,17 +5,11 @@
 
 from django.db import transaction
 
-from apps.quiz.models import Quiz, quizzes
+from apps.quiz.models import Quiz
 from apps.quiz.utils import get_quiz_or_404
 from apps.users.models import TrainingResult
 
 
-def quiz_list_page(request):
-    return {
-        'quizzes': quizzes.iteritems()
-    }
-
-
 def quiz_page(request, quiz_slug):
```

Comment:
@jwalgran Should we squash all this quiz stuff into `apps.home.training` since that is the only place quizzes are referenced from presently?